src/encoder.rs

```diff
@@ -1971,8 +1968,11 @@ pub fn write_tx_blocks<T: Pixel>(
             skip,
             qidx,
             &ac.array,
-            angle_delta.uv,
-            alpha,
+            if angle_delta.uv != 0 {
```

Comment:
The condition `chroma_mode.is_cfl()` matches the choice here, with `IntraParam::Alpha(alpha)` as the positive case.

Reply:
Updated by using chroma_mode.is_cfl().